templates/advantage/subscribe/thank-you.html

```diff
@@ -0,0 +1,25 @@
+{% extends "advantage/base_advantage.html" %}
+
+{% block title %}Ubuntu Advantage for Infrastructure{% endblock %}
+{% block meta_description %}Ubuntu Advantage for Infrastructure offers a single, per-node packaging of the most comprehensive software, security and IaaS support in the industry, with OpenStack support, Kubernetes support included, and Livepatch, Landscape and Extended Security Maintenance to address security and compliance concerns.{% endblock %}
+
+{% block head_extra %}<meta name="robots" content="noindex" />{% endblock %}
+
+{% block content %}
+  <section class="p-strip--suru-topped">
+    <div class="row">
+      <div class="col-12 u-sv3">
+        <h1>Thanks for your purchase</h1>  
+
+        {% if email %}
+          <p class="p-heading--four">We've sent your invoice to {{ email }}</p>
```

Comment:
```suggestion
          <p class="p-heading--four">We’ve sent your invoice to {{ email }}</p>
```
Is there a reason for this paragraph, and the next one, to be `p-heading--four`?